data-es2016plus.js

```diff
@@ -806,7 +806,7 @@ exports.tests = [
           firefox2: false,
           firefox52: true,
           opera10_50: false,
-          safari10_1: true,
+          safari10_1: false,
```

Comment:
Do they work in Safari 11? if so, let's add `safari11: true`. Also, you said to 10.3.1 - is there a 10.3.2 or 10.4 that it does work in? If so, let's add it to environments.json and set *it* to `true` as well.

Reply:
I will test Safari 10, 10.1, 10.2 and so on tomorrow and update the table accordingly 👍

Reply:
After a few tests, both on Safari Mobile and Safari Webview, here are the results:

```
NOT WORKING:
Safari Webview iOS 10.1
Safari Webview iOS 10.2
Safari Webview iOS 10.3.1
Safari Mobile iOS 10.3.1
Safari Webview iOS 11

WORKING:
Safari Mobile iOS 9.3
Safari Mobile iOS 10.1
Safari Mobile iOS 10.2
Safari Mobile iOS 11
```
(couldn't test Safari Webview iOS 9.3 yet)
(Thanks to @titozzz who helped me a lot with the tests ❤️)

I'm not sure what to do with this? Since for the same version of iOS, it sometimes works on Mobile but not on Webview.

Reply:
Hey, any update on this?

Reply:
basically, any result that can’t be represented right now will necessitate the addition of browsers to environments.json.

Reply:
Alright, we will do this with @Titozz tomorrow

Reply:
@Titozzz just tested everything again, and I was mistaken.

Only Safari 10.3 (both webview and mobile) has the bug. Will update the PR

Reply:
Update:

Only **iOS 10.3** has the bug. This is Safari 10.1.

iOS 10, 10.1 and 10.2 have Safari 10.0.

I have update the PR, I think this is good this time, please let me know if not! Thanks